### ISSUE TITLE
ARIA Labels for main navigation

### ISSUE THREAD

**Issue by SteveRyan-ASU** [asuengineering/asu-labs-theme#16]
Include appropriate ARIA labels within the main navigation for the site. Start with `<nav id="menu">`

## Additional information from the report:
HTML5 and WAI-ARIA landmarks for the two different navigations on the page (ASU and sidebar navigation) should have appropriate labels. Review the HTML5 or WAI-ARIA landmarks to make sure the landmark types match the purpose of the sections they are applied to.

## Reference 
(Reference WCAG 2.1, 1.3.1 Info and Relationships)
WCAG 2.1 Score: Level A
Potential User Barrier Level: Low
